src/gurt/tests/SConscript

```diff
@@ -0,0 +1,74 @@
+# (C) Copyright 2020 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+# The Government's rights to use, modify, reproduce, release, perform, display,
+# or disclose this software are subject to the terms of the Apache License as
+# provided in Contract No. 8F-30005.
+# Any reproduction of computer software, computer software documentation, or
+# portions thereof marked with this legend must also reproduce the markings.
+"""Unit tests"""
+
+import os
+import daos_build
+
+TEST_SRC = ['test_gurt.c']
+LIBPATH = [Dir('../')]
```

Comment:
it would be better to add this in the gurt SConscript

genv = env.Clone()
#Add gurt paths here to CPPPATH, LIBPATH, etc
Export('genv')

then here
Import('genv')
...

Reply:
Found that there is no need to configure LIBPATH here. It is already part of env.